doc/source/train/examples/pytorch/pytorch_resnet_finetune.ipynb

```diff
@@ -0,0 +1,776 @@
+      ")\n"
+     ]
+    }
+   ],
+   "source": [
+    "trainer = TorchTrainer(\n",
+    "    train_loop_per_worker=train_loop_per_worker,\n",
+    "    train_loop_config=train_loop_config,\n",
+    "    scaling_config=scaling_config,\n",
+    "    run_config=run_config,\n",
+    ")\n",
+    "\n",
+    "result = trainer.fit()\n",
+    "print(result)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the checkpoint for batch prediction:\n",
+    "\n",
+    " \n",
+    " The metadata and checkpoints have already been saved in the `local_dir` specified in TorchTrainer:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/tmp/ray_results/finetune-resnet/TorchTrainer_4c393_00000_0_2023-03-01_12-40-31/checkpoint_000009\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "['.metadata.pkl', 'dict_checkpoint.pkl', '.is_checkpoint', '.tune_metadata']"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import os\n",
+    "\n",
+    "checkpoint_folder = result.checkpoint.uri.replace(\"file://\", \"\")\n",
+    "print(checkpoint_folder)\n",
+    "os.listdir(checkpoint_folder)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now need to load the trained model and evaluate it on test data. The best model parameters have been saved in `log_dir`. We can use {meth}`TorchCheckpoint.from_directory() <ray.air.checkpoint.Checkpoint.from_directory>` to load the resulting checkpoint from our fine-tuning run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/ray/anaconda3/lib/python3.9/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.\n",
+      "  warnings.warn(\n",
+      "/home/ray/anaconda3/lib/python3.9/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=ResNet50_Weights.IMAGENET1K_V1`. You can also use `weights=ResNet50_Weights.DEFAULT` to get the most up-to-date weights.\n",
+      "  warnings.warn(msg)\n"
+     ]
+    }
+   ],
+   "source": [
+    "checkpoint = TorchCheckpoint.from_directory(checkpoint_folder)\n",
+    "model = checkpoint.get_model(initialize_model())\n",
+    "device = torch.device(\"cuda\")\n"
```

Comment:
We already have an `initialize_method_from_checkpoint` method defined earlier. Let's use that and pass in the `file://` URI.